src/pcre2_jit_compile.c

```diff
@@ -14474,7 +14474,7 @@ if (!check_opcode_types(common, common->start, ccend))
   }
 
 /* Checking flags and updating ovector_start. */
-if (mode == PCRE2_JIT_COMPLETE && (re->flags & PCRE2_LASTSET) != 0 && (re->overall_options & PCRE2_NO_START_OPTIMIZE) == 0)
+if (mode == PCRE2_JIT_COMPLETE && (re->flags & PCRE2_LASTSET) != 0 && (re->optimization_flags & PCRE2_OPTIM_START_OPTIMIZE))
```

Comment:
this will be easier to read if split after the "&&", and yes I know it was like that before, but ...
BTW, the usual syntax aligns it 1 character to the right of the opening parenthesis of `if`

Reply:
Thanks, done.